packages/playground/src/pages/sandbox-settings-project-members-page.tsx

```diff
@@ -1,17 +1,111 @@
-import React from 'react'
-import { Spacer, Text } from '@harnessio/canary'
-import { SandboxLayout, SkeletonList } from '..'
+import React, { useState } from 'react'
+import { Spacer, Text, ListActions, SearchBox, Button } from '@harnessio/canary'
+import { SandboxLayout, SkeletonList, NoData } from '..'
+import { mockMemberData } from '../data/mockMembersData'
+import { MembersList } from '../components/members-list'
+import { PlaygroundListSettings } from '../settings/list-settings'
+import { PaginationComponent } from '../components/pagination'
+
+const filterOptions = [{ name: 'Filter option 1' }, { name: 'Filter option 2' }, { name: 'Filter option 3' }]
+const sortOptions = [{ name: 'Sort option 1' }, { name: 'Sort option 2' }, { name: 'Sort option 3' }]
+
+type MembersProps = {
+  display_name: string
+  role: string
+  email: string
+  avatarUrl?: string
+  timestamp?: string
+}
 
 function SandboxSettingsProjectMembersPage() {
+  const [loadState, setLoadState] = useState('data-loaded')
+
+  const renderMemberListContent = () => {
+    switch (loadState) {
+      case 'loading':
+        return <SkeletonList />
+      case 'data-loaded':
+        return <MembersList members={mockMemberData as MembersProps[]} />
+      case 'no-search-matches':
+        return (
+          <>
+            <Spacer size={10} />
+            <NoData
+              iconName="no-search-magnifying-glass"
+              title="No search results"
+              description={['Check your spelling and filter options,', 'or search for a different keyword.']}
+              primaryButton={{ label: 'Clear search' }}
+              secondaryButton={{ label: 'Clear filters' }}
+            />
+          </>
+        )
+    }
+  }
+
+  const showMembersCounts = () => {
+    switch (loadState) {
+      case 'data-loaded':
+        return ', 30 members'
+      case 'no-data':
+        return ', 0 members'
+      case 'no-search-matches':
+        return ', 30 members'
+      default:
+        return ''
+    }
+  }
```

Comment:
Not sure what this is doing? Are we using it somewhere?

Reply:
it's about showing accurate numbers of members in the title of team. In the different status, we show different numbers. For example, it will show the current numbers of members if data-loaded, if there's no search data, 
<img width="1114" alt="Screenshot 2024-10-16 at 1 03 12 PM" src="https://github.com/user-attachments/assets/af6204b3-6ee4-4c92-be5f-e8c77b905c12">
I can update the new functions only in the component since no-data not related to this count anymore

Reply:
fixed